graviti/manager/draft.py

```diff
@@ -139,24 +139,22 @@ def edit(self, title: Optional[str] = None, description: Optional[str] = None) -
             title=title,
             description=description,
         )
-        if title is not None:
-            self.title = title
-        if description is not None:
-            self.description = description
-        # TODO: update the draft.updated_at
+        self.title = response["title"]
+        self.description = response["description"]
+        self.updated_at = response["updated_at"]
 
     def close(self) -> None:
         """Close the draft."""
-        update_draft(
+        response = update_draft(
```

Comment:
should it be `draft_info` or use `response` for all the api return

Reply:
should use response for all the api return